src/Deprecated/Conversion/Microsoft.Build.Conversion.csproj

```diff
@@ -15,7 +15,6 @@
     <Compile Include="AssemblyInfo.cs" />
     <Compile Include="AssemblyResources.cs" />
     <Compile Include="Constants.cs" />
-    <Compile Include="FxCopExclusions\Microsoft.Build.Conversion.Suppressions.cs" />
```

Comment:
Do you think it's worth making this change in `Deprecated/`? I usually try to leave that alone. On the other hand you already did it and it should be fine . . .

Reply:
I would lean towards making the change. Although not under active development the code still benefits from cleanup (a bit faster git operations, a bit faster build).